### PR TITLE
Fix too strong conditions in tests

### PR DESCRIPTION
**Pull request by MathisGD**

### CODE REVIEW

File: test/TestMath.sol [hunk 1]
```diff
@@ -83,14 +83,12 @@ contract TestMath is Test {
 
     function testDivUpWhenNumLargerAndDivisible(uint256 x, uint256 y) public {
         vm.assume(y > 0);
-        vm.assume(x > y);
```
Comment (QGarchery) [inline test/TestMath.sol:86]: We can now change the name of the function

File: test/TestMath.sol [hunk 2]
```diff
@@ -83,14 +83,12 @@ contract TestMath is Test {
...
         vm.assume(x % y == 0);
         assertEq(Math.divUp(x, y), x / y);
     }
 
     function testDivUpWhenNumLargerAndNotDivisible(uint256 x, uint256 y) public {
         vm.assume(y > 0);
```
Comment (QGarchery) [inline test/TestMath.sol:91]: same

**Review by QGarchery — APPROVED** [review]

File: test/TestMath.sol [hunk 3]
```diff
@@ -70,27 +70,25 @@ contract TestMath is Test {
...
 
-    function testDivUpWhenNumLargerAndDivisible(uint256 x, uint256 y) public {
+    function testDivUpDivisible(uint256 x, uint256 y) public {
         vm.assume(y > 0);
-        vm.assume(x > y);
         vm.assume(x % y == 0);
         assertEq(Math.divUp(x, y), x / y);
     }
 
-    function testDivUpWhenNumLargerAndNotDivisible(uint256 x, uint256 y) public {
```
Comment (Rubilmax) [inline test/TestMath.sol:91]: I know test names are large but it makes reading a test report & the code easier by giving a clear context
I don't think it's the case for `testDivUpNotDivisible` which does not indicate whether it's the numerator or the denominator which is not divisible

I would recommend `testDivUpNumNotDivisible` and `testDivUpNumDivisible`, if making test names shorter is meaningful

**Review by Rubilmax — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]
LGTM

File: test/TestMath.sol (hunk 3 above)
Comment (MathisGD) [reply test/TestMath.sol:91]: >which does not indicate whether it's the numerator or the denominator which is not divisible
this is implicit

But I can re-add that if you want!

Comment (MathisGD) [reply test/TestMath.sol:91]: >which does not indicate whether it's the numerator or the denominator which is not divisible
this is implicit

But I can re-add that if you want!

**Pull request merged**